chromadb/api/models/CollectionCommon.py

```diff
@@ -181,7 +181,7 @@ def _unpack_embedding_set(
 
     def _validate_embedding_set(
         self,
-        ids: IDs,
+        ids: Optional[IDs],
```

Comment:
I don't think `ids` can be `Optional` here, since `validate_ids` fails on `None`

Reply:
true. however, if we are generating ids on the server level, ids should be allowed to be optional here